themes/wporg-support/searchform.php

```diff
@@ -20,7 +20,7 @@
 		$placeholder = _x( 'Search this forum', 'placeholder', 'wporg-forums' );
 		$project     = wporg_support_get_compat_object();
 	} else {
-		$placeholder = _x( 'Search forums', 'placeholder', 'wporg-forums' );
+		$placeholder = _x( 'Search support resources', 'placeholder', 'wporg-forums' );
```

Comment:
'user documentations' might be better. But not a blocker, we can change these later.

Reply:
I went with resources for now, as the current setup will use Google to search wordpress.org/support and thus include everything on that subsite at the moment, so I wanted the description to be representative of that.